src/components/PostItem/PostItem.tsx

```diff
@@ -11,13 +12,17 @@ type PostItemProps = {
   title: string;
 };
 
-export const PostItem = ({ id, title }: PostItemProps) => (
-  <tr>
-    <StyledRowId>{id}</StyledRowId>
+export const PostItem = ({ id, title }: PostItemProps) => {
+  return (
+    <tr>
+      <StyledRowId>{id}</StyledRowId>
 
-    <td>{title}</td>
-    <td>
-      <StyledCustomDetailButton>Detail</StyledCustomDetailButton>
-    </td>
-  </tr>
-);
+      <td>{title}</td>
+      <td>
+        <CustomDetailButton url={`${appRoutesList.postDetail}/${id}`}>
```

Comment:
všechny URL prosím vyextrahuj do jednoho souboru, aby se neskládali stringy v šablonách, ale bylo jedno místo, odkud se to ovládá a aby se udělala abstrakce nad spojováním stringů v komponentách